VotingApplication/VotingApplication.Web/Api/Controllers/ManageController.cs

```diff
@@ -217,7 +217,7 @@ private void SendInvitation(Guid UUID, Token token)
                 return;
             }
 
-            string message = String.Join("\n\n", new List<string>() { "You've been invited to a poll on Pollster",
+            string message = String.Join("\n\n", new List<string>() { "You've been invited to Vote On " + pollQuestion,
```

Comment:
Should this be something more along the lines of:

```javascript
"You've been invited to Vote On : \"" + pollQuestion + "\"",
"Have your say at " + hostUri + "/Poll/#/Vote/" + UUID + "/" + token.TokenGuid });
```

So the poll question doesn't run into the message.